health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -62,12 +65,12 @@ const UserDownload = () => {
             <Card type="secondary" >
                 <HeaderComp title="DOWNLOAD_USER_DATA" styles={{ color: "black" }} />
                 <TextBlock body={t("DOWNLOAD_DESC")} />
-                    {data?.ResourceDetails &&
-                        [...data?.ResourceDetails].reverse().map((item, index) => {
-                            let fileName = item?.additionalDetails?.fileName || `FileNo${item?.processedFilestoreId?.slice(0, 4) || ''}`;
-                            
+                {data?.ResourceDetails &&
+                    [...data?.ResourceDetails].reverse().map((item, index) => {
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)